pages/db/migrate/20150720155305_update_slug_and_title_in_refinery_page_parts.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateSlugAndTitleInRefineryPageParts < ActiveRecord::Migration
+  def change
+    if defined?(::Refinery::PagePart)
+      Refinery::PagePart.all.each do |pp|
+        pp.title ||= pp.slug
+        pp.slug = pp.slug.downcase.gsub(" ", "_")
+        pp.save!
+      end
+    end
+  end
```

Comment:
Actually, this won't work.  Can you use the style from https://github.com/refinery/refinerycms/commit/8f56bf47a7ecd6d348b26d9531ff5e4eb03d5f68 where we use begin + rescue instead? This ensures that the classes at least try to be loaded before declaring that they don't exist as `defined?` doesn't try.